apps/dashboard/app/page.tsx

```diff
@@ -1,18 +1,53 @@
 import { getServerSession } from "next-auth"
 
+import { Box } from "@mui/joy"
+
 import { authOptions } from "./api/auth/[...nextauth]/route"
 
 import PriceContainer from "@/components/price-container/price-container"
 import ContentContainer from "@/components/content-container"
+import { fetchAllTransactionsByCount } from "@/lib/get-all-transactions"
+import TransactionChart from "@/components/chart"
 
 export default async function Home() {
   const session = await getServerSession(authOptions)
+
+  const token = session?.accessToken
+  if (!token || typeof token !== "string") {
+    throw new Error("invalid token")
+  }
+
+  const btcWallet = session.userData.data.me?.defaultAccount.wallets.find(
+    (wallet) => wallet.walletCurrency === "BTC",
+  )
+  const usdWallet = session.userData.data.me?.defaultAccount.wallets.find(
+    (wallet) => wallet.walletCurrency === "USD",
+  )
+
+  const response = await fetchAllTransactionsByCount({
+    token,
+    fetchCount: 5, // Maximum fetch rotation count if set to 5; the API will be called a maximum of 5 times, which gives 500 transactions.
```

Comment:
magic value should be documented with a `const`

Reply:
it;s not very intuitive that each count is 100 transactions. might be better to have something like `txCount = 500`. this would make the comment unecessary

Reply:
> it;s not very intuitive that each count is 100 transactions. might be better to have something like `txCount = 500`. this would make the comment unecessary

For the current implementation, we don't have to do any extra processing since the maximum number of transactions per request is 100. Therefore, we just run it a maximum of 5 times, which gives us 500, with no slicing or other processing required. But, i agree that this might be confusing for others, so I've updated it to take the transaction count, and now we are taking `500` as input for 500 transactions.